README.md

```diff
@@ -26,7 +26,18 @@ You must have Docker Desktop installed for this to work, as the environment is d
 1. Once that has completed, type `docker compose up db -d`
 1. Once that has completed, type `docker compose run --rm web bundle exec rails db:setup`
 1. Once that has completed, type `docker compose up -d`
-1. In your browser, navigate to `https://localhost:3000`
+1. In your browser, navigate to `http://localhost:3000`
+
+### Other helpful commands
+1. To stop the app, type `docker compose stop`
```

Comment:
Or `docker compose down`?

Reply:
Yeah, `down` would have been better. They achieve the same thing though